Tests/GitHubReactions.Tests.ps1

```diff
@@ -26,14 +26,18 @@ try
         BeforeAll {
             $repo = New-GitHubRepository -RepositoryName ([Guid]::NewGuid().Guid) -AutoInit
             $issue = New-GitHubIssue -Uri $repo.svn_url -Title $defaultIssueTitle
+            $issueComment = $issue | New-GitHubIssueComment -Body "Foo"
+
+            # To get rid of linting issue saying this variable isn't used.
+            $issueComment.CommentId
```

Comment:
Please don't do this.  Refer to the other test files.  We've standardized on suppressing that SA rule and enabled strict mode within `Tests/Common.ps1`.

Reply:
fixed